### PR TITLE
Use correct version of Antrea image in tests

### PR DESCRIPTION
**Pull request by antoninbas**

### CODE REVIEW

**Review by dougsland — APPROVED** [review]

File: lib/antrea.sh [hunk 1]
```diff
@@ -25,6 +25,8 @@ source "${DIR_LIBS}/kind.sh"
 # shellcheck source=./common.sh
 source "${DIR_LIBS}/common.sh"
 
+ANTREA_IMG_NAME="projects.registry.vmware.com/antrea/antrea-ubuntu"
```
Comment (dougsland) [inline lib/antrea.sh:28]: Probably a good candidate for config.sh -> Antrea? 
https://github.com/K8sbykeshed/k8s-local-dev/blob/main/lib/config.sh#L49-L55

File: lib/antrea.sh [hunk 2]
```diff
@@ -55,7 +57,10 @@ function antrea_create_cluster() {
     ##################################################################
     # Description:                                                   #
     #   create the kind cluster                                      #
+    #                                                                #
+    #   Args: ${1} - version                                         #
     ##################################################################
+    version="${1}"
```
Comment (dougsland) [inline lib/antrea.sh:63]: @antoninbas I am wondering if users do: ./k8s-local-dev antrea (not specifying the version)
shouldn't we do a fallback to default version or latest? 

https://github.com/K8sbykeshed/k8s-local-dev/blob/main/lib/config.sh#L50-L52

Example:
if ${1} not defined..
   use ANTREA_VERSION
fi

**Review by dougsland — COMMENTED** [review]
@antoninbas I already merged the patch because we had failures but I have few suggestions for new patches. Please let me know if you are interested in address them :) thanks again!

File: lib/antrea.sh (hunk 1 above)
Comment (antoninbas) [reply lib/antrea.sh:28]: Yes I can open a new PR for that

File: lib/antrea.sh (hunk 2 above)
Comment (antoninbas) [reply lib/antrea.sh:63]: That doesn't seem needed, `config.sh` already takes care of that

**Pull request merged**